coalib/results/Diff.py

```diff
@@ -70,6 +71,24 @@ def from_string_arrays(cls, file_array_1, file_array_2, rename=False):
         return result
 
     @classmethod
+    def from_unidiff(cls, diff_array):
+        """
+        Creates a Diff object from given unified diff.
+
+        :param diff_array: list of lines of unified diff.
```

Comment:
Maybe we should keep them as arrays just for the sake of consistency and readability because rest of the tests define arrays (even in `LinterTest.py`), just a suggestion though :)